content/blog/insights-cloud-account-discovery/index.md

```diff
@@ -0,0 +1,95 @@
+
+Account Discovery starts with the new Accounts page to provide a list of all accounts created along with the current status of the latest account scans. The Accounts page is the single place to go to manage all aspects of the Insights Account Discovery process.
+
+{{< video title="Accounts" src="accounts.mp4" autoplay="true" loop="true" >}}
+
+### Account Creation
+
+From this page you can create new Accounts to provide the configuration and credentials needed for Insights to regularly scan and manage your infrastructure. Once you create a new top level Account, Insights will automatically create child accounts, based on the underlying platform model, for each group you enabled. These child accounts enable you to control the discovery behavior for each group separately. For example, AWS enables you to divide you infrastructure into regions and Insights will create separate child accounts for each region you specify.
+
+{{% notes type="info" %}}
+Account Discovery leverages Pulumi ESC to enter and manage the credentials needed for the Discovery process to find and read all of the infrastructure resources. By relying on ESC, Insights aligns with enterprise best practices for creating and handling application secrets.
+{{% /notes %}}
+
+{{< video title="Create Account" src="create-account.mp4" autoplay="true" loop="true" >}}
+
+### Understanding Your Infrastructure
+
+Once each of your Accounts have started scanning, your resources will be displayed in the Insights Resources Explorer along with all Infrastructure as Code resources that you have imported. The Resources Explorer and Resources Search enables you to ask questions and factor your resources into logical groupings that significantly improves the process for managing your infrastructure and getting answers to key questions needed for your projects. For example, you can ask the AI Resource Search to 'Find all public IP addresses' and any other question you might have about your infrastructure.
+
+{{< video title="Discovered Resources" src="resources.mp4" autoplay="true" loop="true" >}}
+
+### Conclusion
+
+The addition of Account Discovery significantly expands the scope of Pulumi Insights. You can now leverage the capabilities of Insights 2.0, not just for your Pulumi IaC managed resources, but for all resources in your infrastructure regardless of how it is managed. With Pulumi Copilot and Resource Search, you are able to gain insights and ask questions about your infrastructure that would otherwise be challenging to answer, saving time and providing the critical information needed.
+
+For additional information about Pulumi Insights please refer to: [Pulumi Insights](https://www.pulumi.com/product/pulumi-insights/)
```

Comment:
Should we have a CTA to sign-up? Should it link directly to the "Register a New Account" page?

Reply:
The challenge is that Insights is not available to the Individual tier so the CTA would not be relevant to a portion of the readers. We are going to add an upsell screen for individual users that click on Accounts in the left nav, but today Accounts is hidden for individual users.